cuckoo/table.go

```diff
@@ -118,6 +79,17 @@ func (t *Table) GetNumElements() int {
 	return result
 }
 
+// Bucket returns the bucket in a table that the Item is in, if it is in the table.
+// -1 otherwise
+func (t *Table) Bucket(item *Item) (uint64, error) {
+	if t.isInBucket(item.Bucket1, item) {
+		return item.Bucket1, nil
+	} else if t.isInBucket(item.Bucket2, item) {
+		return item.Bucket2, nil
+	}
+	return 0, fmt.Errorf("%v.Bucket(%v): item not in table", t.name, item)
```

Comment:
I've been consistently using uint64 everywhere. I could change this method to return an int64, but in terms of consistency, do you think all of cuckoo should be int64 or even all of Talek?
Thoughts?